host.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "2.0",
+    "logging": {
+        "applicationInsights": {

```

Comment:
I don't think we get Application Insights with the basic Function App I set up, can look into upgrading the Function App at some point.

Reply:
I see. This was just the defaults when generating a function through azure function commands.